### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaybee84**

### CODE REVIEW

File: R/mod_summary_snapshot.R [hunk 1]
```diff
@@ -53,7 +46,7 @@ mod_summary_snapshot_ui <- function(id){
             shiny::fluidRow(
               shinydashboard::infoBoxOutput(ns('box1'), width = 3),
               shinydashboard::infoBoxOutput(ns('box2'), width = 3),
-              shinydashboard::infoBoxOutput(ns('box3'), width = 3),
+              # shinydashboard::infoBoxOutput(ns('box3'), width = 3),
```
Comment (jaybee84) [inline R/mod_summary_snapshot.R]: Can we replace the sample count with dataset count?

**Review by jaybee84 — COMMENTED** [review]
Since this PR was started by me, GH wont let me request changes ! :D 

So here are a few comments  for changes:
1. lets add dataset count in box 3 instead of sample count
2. The consortium activity plot is not filling up the box and skewed to one side of the box
3.  Resources generated Y-axis needs to be parsed by the themes i.e. if it is a list, each item in the list should be one item on Y-axis. I think this will mean that some datasets will be duplicated in the different themes. That should be okay for now.

**Comment by andrewelamb** [comment]
"lets add dataset count in box 3 instead of sample count" 

I'm using datasetId as an id for files. If we add datasets we will have two boxes both showing the same number.

**Comment by andrewelamb** [comment]
"The consortium activity plot is not filling up the box and skewed to one side of the box"

Can you clarify?

"Resources generated Y-axis needs to be parsed by the themes i.e. if it is a list, each item in the list should be one item on Y-axis. I think this will mean that some datasets will be duplicated in the different themes. That should be okay for now."

This has been fixed.

**Comment by jaybee84** [comment]
Re: datasets :
Yeah... I realized that the "Studies" box is also showing dataset count. Ideally the "Studies" box should show Grants count, and "Files" box should show dataset count

**Comment by jaybee84** [comment]
Re: dataset count
Yeah.. I realized it later :) ... no need to add another box

**Comment by andrewelamb** [comment]
"Re: datasets :
Yeah... I realized that the "Studies" box is also showing dataset count. Ideally the "Studies" box should show Grants count, and "Files" box should show dataset count"

Currently:
box1: "Studies" using grantId
box2: "Files" using datasetId
box3: "Datasets" using datasetId
box4: publications using publicationId

**Comment by jaybee84** [comment]
Re: "The consortium activity plot is not filling up the box and skewed to one side of the box"
image

**Comment by andrewelamb** [comment]
I see about the plots.

IIRC ggplotly is causing the mess up here and I was unable to fix this before.

**Comment by jaybee84** [comment]
Re: dataset count 
Three boxes seem right.. My earlier comment was misinformed ... sorry for the confusion :)

**Comment by jaybee84** [comment]
Re: Box 3 /Dataset count

I was looking through the CSBC Portal tables a bit more and found this table with all the files in the portal: https://www.synapse.org/#!Synapse:syn9630847/tables/ 
Maybe we can use this for Box 2 : "Files" ?

**Comment by andrewelamb** [comment]
Maybe this is a stupid question, why aren't we using the files table for our files like we do in NF, instead of the datasets table?

**Comment by jaybee84** [comment]
Not a stupid question at all! There are mainly 2 reasons:
1. I did not find the table earlier and assumed that CSBC data is organized into datasets only :) (my mistake)
2. The datasets seem to be the ones that are externally linked which mimics how the INCLUDE data would look like. I haven't looked deeply into the files table, but if you see that majority of the files are also externally linked (i.e. linking to GEO or SRA), we can go ahead and use that table.

**Comment by andrewelamb** [comment]
I  am now using the files table instead of the datasets table. The onyl issue is that the Theme column in the files table is all NA's

**Comment by jaybee84** [comment]
@andrewelamb  just checking if this PR is still being worked on or ready for review?

**Comment by andrewelamb** [comment]
@jaybee84 yep, Its ready for review!

**Pull request merged**